conf/app.conf

```
@@ -9,3 +9,4 @@ mongo_db = ${CUENTAS_CONTABLES_CRUD_DB_NAME}
 mongo_user = ${CUENTAS_CONTABLES_CRUD_DB_USER}
 mongo_pass = ${CUENTAS_CONTABLES_CRUD_DB_PASS}
 mongo_db_auth = ${CUENTAS_CONTABLES_CRUD_DB_AUTH}
+mongo_port = ${CUENTAS_CONTABLES_CRUD_DB_PORT}
```

(En general) Dejar vacía, sin espacios ni nada, la última línea en archivos de texto
```suggestion
mongo_port = ${CUENTAS_CONTABLES_CRUD_DB_PORT}

```